sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -62,6 +62,8 @@ namespace Azure { namespace Core { namespace Http {
      * @remark More about this option:
      * https://curl.se/libcurl/c/CURLOPT_CAINFO_BLOB.html
      *
+     * @warning Requires libcurl >= 7.44.0
```

Comment:
Does it make sense to conditionally remove this field if libcurl < 7.44?

That way there is no way for someone to accidentally try to use it.

Reply:
We could, but that would be a breaking change. I think we should throw an exception if the user sets this, at runtime, so at least they know.

Reply:
I was worried about conditional surface area.  I'm not married to my solution though

Reply:
It's only technically a breaking change - if someone on curl < 7.44 is using this field, it doesn't do anything, and honestly I *want* them to break in that situation, because their configuration isn't working.